packages/slice-machine/lib/models/common/widgets/Group/ListItem/index.js

```diff
@@ -59,10 +59,15 @@ const CustomListItem = ({
     setSelectMode(false);
   };
 
+  /* eslint-disable */
   const getFieldMockConfig = ({ apiId }) => {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument
-    return CustomTypeMockConfig.getFieldMockConfig(mockConfig, apiId);
+    return CustomTypeMockConfig.getGroupFieldMockConfig(
```

Comment:
unless i'm wrong, that function doesn't exists, this should instant fail.

Reply:
it does

Reply:
I created it myself, that's what is supposed to fix the group issue.

Reply:
but you're right I don't see it anymore that's weird!

Reply:
Yeah, I checked multiple times to see if I was wrong haha